### ISSUE TITLE
Implementar visualização "apagada" no card qdo a missão estiver expirada

### ISSUE THREAD

**Issue by ricardopoppi** [unidoscontraacorrupcao/besouro-app#37]
Conforme wireframes: tela de "[minhas missões](https://invis.io/KYMC4N035H9#/302750833_Home_suas_Missoes)" e o [ícone do relógio tocando](https://drive.google.com/drive/folders/1_-5zo9m7dRiGfLML5ZInQOgJnqomBaIV?ogsrc=32) já está na pasta do drive

**Comment by davidCarlos** [comment]
@ricardopoppi já tem tela no invision pra isso?

**Comment by ricardopoppi** [comment]
@davidCarlos sim! comentei no corpo da issue: https://github.com/unidoscontraacorrupcao/besouro-app/issues/37#issue-337677292
